deployments/common/ap-policy-definition.yaml

```diff
@@ -0,0 +1,874 @@
+                  type: string
+                enablePassiveMode:
+                  type: boolean
+                enforcementMode:
+                  enum:
+                  - transparent
+                  - blocking
+                  type: string
+                filetypes:
+                  items:
+                    properties:
+                      allowed:
+                        type: boolean
+                      checkPostDataLength:
+                        type: boolean
+                      checkQueryStringLength:
+                        type: boolean
+                      checkRequestLength:
+                        type: boolean
+                      checkUrlLength:
+                        type: boolean
+                      name:
+                        type: string
+                      postDataLength:
+                        type: integer
+                      queryStringLength:
+                        type: integer
+                      requestLength:
+                        type: integer
+                      responseCheck:
+                        type: boolean
+                      type:
+                        enum:
+                        - explicit
+                        - wildcard
+                        type: string
+                      urlLength:
+                        type: integer
+                    type: object
+                  type: array
+                fullPath:
+                  type: string
+                general:
+                  properties:
+                    allowedResponseCodes:
+                      items:
+                        format: int32
+                        maximum: 999
+                        minimum: 100
+                        type: integer
+                      type: array
+                    customXffHeaders:
+                      items:
+                        type: string
+                      type: array
+                    enforcementReadinessPeriod:
+                      type: integer
+                    maskCreditCardNumbersInRequest:
+                      type: boolean
+                    trustXff:
+                      type: boolean
+                  type: object
+                header-settings:
+                  properties:
+                    maximumHttpHeaderLength:
+                      pattern: any|\d+
+                      type: string
+                  type: object
+                headers:
+                  items:
+                    properties:
+                      base64Decoding:
+                        type: boolean
+                      checkSignatures:
+                        type: boolean
+                      htmlNormalization:
+                        type: boolean
+                      id:
+                        type: string
+                      kind:
+                        type: string
+                      lastUpdateMicros:
+                        type: integer
```

Comment:
# Fields that are from BIG-IP REST that are invalid or irrelevant for Declarative
- id
- kind
- lastUpdateMicros
- selfLink